examples/typescript/evm/importAccount.ts

```diff
@@ -0,0 +1,11 @@
+// Usage: pnpm tsx evm/importAccount.ts
+
+import { CdpClient } from "@coinbase/cdp-sdk";
+import "dotenv/config";
+
+const cdp = new CdpClient();
+const account = await cdp.evm.importAccount({
+  privateKey: "0x123456",
```

Comment:
Let's use a real generated private key so that the example works!

Reply:
You suggest I randomly generate one offline and just hardcode it here?

Reply:
Also reminder to run the formatter!

Reply:
Sorry, meant to include the snippet in the original message^ @marcin-cb